test/e2e/flowaggregator_test.go

```diff
@@ -195,12 +186,20 @@ func checkRecordsForFlows(t *testing.T, data *TestData, srcIP string, dstIP stri
 		if strings.Contains(record, srcIP) && strings.Contains(record, dstIP) {
 			dataRecordsCount = dataRecordsCount + 1
 			// Check if records have both Pod name and Pod namespace or not.
-			if !strings.Contains(record, "perftest-a") {
-				t.Errorf("Records with srcIP does not have Pod name")
-			}
-			if !strings.Contains(record, "perftest-b") && isIntraNode {
-				t.Errorf("Records with dstIP does not have Pod name")
+			if isIntraNode {
+				if !strings.Contains(record, "perftest-a") {
+					t.Errorf("Records with srcIP does not have Pod name")
+				}
+				if !strings.Contains(record, "perftest-b") {
+					t.Errorf("Records with dstIP does not have Pod name")
+				}
+			} else {
+				// Flow record comes from destination node may contain incomplete pod name information
```

Comment:
s/Flow record comes/Flow record coming

However, I thought that the aggregator ensures no incomplete information in records? @srikartati

Reply:
May be this comment was copied from Flow Exporter code. We should have complete information with flow aggregator.

Reply:
Actually I added this if-condition because sometimes aggregator doesn't have complete information in the Kind cluster e2e test. Not sure go-ipfix v0.4.2 solved this issue or not, will try again after PR https://github.com/vmware-tanzu/antrea/pull/1676 merged.

Reply:
go-ipfix v0.4.2 should resolve this.. we introduced readyToSend field in aggregator process to wait until we receive the records from source and destination node.